source/includes/connection-snippets/x509-snippets/python-connection.py

```diff
@@ -0,0 +1,14 @@
+# begin x509 connection
+from pymongo import MongoClient
+
+uri = "mongodb+srv://<cluster-url>/?authMechanism=MONGODB-X509"
+client = MongoClient(uri,
+                     tls=True,
+                     tlsCertificateKeyFile='/etc/certs/mongodb/client.pem',
+                     tlsAllowInvalidCertificates=False)
```

Comment:
AFAIK, `tlsAllowInvalidCertificates` should already default to `false` in all drivers, so this might be redundant. I also didn't see this in any other language examples.

Reply:
Sorry, `tlsAllowInvalidCertificates` should not be in there. I've removed it now